### ISSUE TITLE
Customer Details in Account Receivable PDF

### ISSUE THREAD

**Issue by maveezmh** [frappe/erpnext#35030]
### Information about bug

When you generate accounts Receivable/payable report  or Statement of Accounts for a customer/supplier. we need to show customer details in  the generated PDF. but there is no wat to do that. we need to create custom report apart from accounts receivable/payable. and it will take away lots of other features. 

is there any Way to add customer reports in those generated PDF?

![Screenshot 2023-04-25 at 12-59-06 e92abc8a-e900-4f5c-ba10-4eca7f76a8db](https://user-images.githubusercontent.com/112148467/234227646-8ec7cfda-c964-4acb-847c-ce27d7f37da3.png)

### Module

accounts, buying, selling

### Version

All versions

### Installation method

FrappeCloud

**Comment by dj12djdjs** [comment]
I agree, there should be some further customization for these accounting reports.

Especially, reports that have to get mailed to the customer. It is important to show information such as billing address.

**Comment by maveezmh** [comment]
@dj12djdjs Lot of customers complain about this. and right now the only way to show that is by creating custom reports. which is just a duplicate of accounts receivable. and the custom report wont serve the complete purpose too.

**Comment by shahbazafzal1** [comment]
Yes This is a big issue in reports still Unsolved we need to add customer address and customer name in bold manner but find no way except to to custom, report with lot of qurries

**Comment by DipenFrappe** [comment]
Please comment what all information required?

**Comment by maveezmh** [comment]
> Please comment what all information required?

- option to set default letterhead for reports, 
- customer Name instead of ID
- Customer TRN/GST/VAT number
- Default customer Address
- also need option to group the receivables by receivable account

**Comment by nabinhait** [comment]
Reference Print Format: https://github.com/Cecypo-Tech/frappe_customer_statements

**Comment by maveezmh** [comment]
> Reference Print Format: https://github.com/Cecypo-Tech/frappe_customer_statements

its good solution... i gues for account receivable and payable its not working. its only working for processing statement of accounts if we can bring this to acc payable and receivable. it will be really helpful.
